models/buttonRoleConfig.js

```diff
@@ -0,0 +1,14 @@
+// models/buttonRoleConfig.js
+module.exports = (client, Sequelize) => {
```

Comment:
You would need to set up a guildID to make this work -- a general buttonId across all discords would not work